rossum/user.py

```diff
@@ -88,10 +88,11 @@ def change_command(
         return
 
     data: Dict[str, Any] = {}
-    if password is not None:
-        data["password"] = password
 
     with RossumClient(context=ctx.obj) as rossum:
+        if password is not None:
```

Comment:
Consider moving this check to the end of the function. This way, we won't need the function to end prematurely, but after the queue patch is done. The mutual exclusivity can be then removed.